js/about/contributionStatement.js

```diff
@@ -160,10 +174,19 @@ class ContributionStatement extends ImmutableComponent {
   }
 
   get ContributionStatementHeader () {
+    const imgStyle = StyleSheet.create({
```

Comment:
method to transform image to base64 is a thenable, so I had to pass result to a state, and `styles` is not bound to the component, so I can't call `this.state` there. That's why I had to create a separate stylesheet for each image